core/src/main/scala/org/apache/spark/sql/TiExtensions.scala

```diff
@@ -13,9 +13,9 @@ class TiExtensions extends (SparkSessionExtensions => Unit) {
   }
 
   override def apply(e: SparkSessionExtensions): Unit = {
-    e.injectParser(ReflectionUtil.newTiParser(getOrCreateTiContext))
-    e.injectResolutionRule(ReflectionUtil.newTiDDLRule(getOrCreateTiContext))
-    e.injectResolutionRule(ReflectionUtil.newTiResolutionRule(getOrCreateTiContext))
+    //e.injectParser(ReflectionUtil.newTiParser(getOrCreateTiContext))
```

Comment:
would u pls add `if else` here to choose from `v1` and `v2`?
maybe add a config switch `use_3.0_catalog`?